spec/03-plugins/26-prometheus/05-metrics_spec.lua

```diff
@@ -0,0 +1,143 @@
+local helpers = require "spec.helpers"    -- hard dependency
+
+
+local ngx = ngx
+
+
+local fixtures = {
+    dns_mock = helpers.dns_mock.new({ mocks_only = true, }),
```

Comment:
Spacing: we usually indent Lua files with 2 spaces, not 4.

(I know this is inconsistent with the spacing used on tests inside `t/`, but those are the exception)